doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -168,7 +168,9 @@ This section provides detailed lists of changes by :ref:`application <applicatio
 Asset Tracker v2
 ----------------
 
-  * Removed nRF7002 EK DTC overlay file ``nrf91xxdk_with_nrf7002ek.overlay`` because UART1 is disabled through the shield configuration.
+* Added the :kconfig:option:`CONFIG_DATA_SAMPLE_WIFI_DEFAULT` Kconfig option to configure if Wi-Fi APs are included in sample requests by default.
+
+* Removed nRF7002 EK DTC overlay file ``nrf91xxdk_with_nrf7002ek.overlay`` because UART1 is disabled through the shield configuration.
```

Comment:
This sentence came in from PR https://github.com/nrfconnect/sdk-nrf/pull/13445 and is in 3 other places of the release note.

Reply:
And do we want `:file:` tag for a file that's removed from the repository?

Reply:
Yep, I was sloppy with the previous review, sorry. The :file: tag does not really have much functionality and does not have a big impact on anything. It is just for formatting and double backticks would do the same trick.

Reply:
This PR only changed the indentation, otherwise this line is unchanged. But we can of course do some fine tuning, if necessary. 😄

Reply:
> Yep, I was sloppy with the previous review, sorry. The :file: tag does not really have much functionality and does not have a big impact on anything. It is just for formatting and double backticks would do the same trick.

Ok, I was thinking about :file: tag originally but didn't do it since the doc generation results haven't been properly available although they should be now.

Reply:
Updated all four occurrences.